guides/release/components/triggering-changes-with-actions.md

```diff
@@ -103,34 +103,37 @@ in the child component we will implement the logic to confirm that the user want
 
 ```javascript {data-filename=app/components/button-with-confirmation.js}
 import Component from '@ember/component';
```

Comment:
@locks @pzuraq if we use GC, should the import be something else? Do we know this yet?

Reply:
Go with `import Component from '@glimmer/component';` for now!